app/src/main/java/com/sunasterisk/moviedb_51/ui/details/MovieDetailsFragment.kt

```diff
@@ -0,0 +1,14 @@
+package com.sunasterisk.moviedb_51.ui.details
+
+import androidx.core.os.bundleOf
+import androidx.fragment.app.Fragment
+import com.sunasterisk.moviedb_51.utils.Constant
+
+class MovieDetailsFragment : Fragment() {
+
+    companion object {
+        fun getInstance(id: Int, title: String) = MovieDetailsFragment().apply {
+            arguments = bundleOf(Constant.BASE_VALUE to id, Constant.BASE_TITLE to title)
```

Comment:
BASE VALUE, BASE TITLE có dùng ở đâu nữa không ? 
Xem lại cách đặt tên 
https://github.com/framgia/coding-standards/blob/master/eng/android/codingstyleguide.md#210-string-constants-naming-and-values